scripts/exglobal_fetch.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import os
+
+from fetch import Fetch
+from wxflow import AttrDict, Logger, cast_strdict_as_dtypedict, logit
+
+# initialize root logger
+logger = Logger(level=os.environ.get("LOGGING_LEVEL", "DEBUG"), colored_log=True)
+
+
+@logit(logger)
+def main():
+
+    config = cast_strdict_as_dtypedict(os.environ)
+
+    # Instantiate the Fetch object
+    fetch = Fetch(config)
+
+    # Pull out all the configuration keys needed to run the fetch step
+    keys = ['current_cycle', 'RUN', 'PDY', 'PARMgfs', 'PSLOT', 'ROTDIR', 'fetch_yaml', 'ATARDIR', 'ntiles']
```

Comment:
Now that you have a `FETCHDIR`, it should replace `ATARDIR` in the fetch job.
```suggestion
    keys = ['current_cycle', 'RUN', 'PDY', 'PARMgfs', 'PSLOT', 'ROTDIR', 'fetch_yaml', 'FETCHDIR', 'ntiles']
```

Reply:
Right - done.